LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/IoTHubConnector.cs

```diff
@@ -151,10 +151,17 @@ private void CreateDeviceClient()
                 {
                     string partConnection = createIoTHubConnectionString();
                     string deviceConnectionStr = $"{partConnection}DeviceId={DevEUI};SharedAccessKey={PrimaryKey}";
-
-                    deviceClient = DeviceClient.CreateFromConnectionString(deviceConnectionStr, TransportType.Amqp_Tcp_Only);
-
-                   
+                    
+                    if (configuration.LeafDeviceProtocol.ToUpper() == "AMQP_TCP_ONLY")
+                    {
+                        deviceClient = DeviceClient.CreateFromConnectionString(deviceConnectionStr, TransportType.Amqp_Tcp_Only);
+                        Logger.Log(DevEUI, $"device client created using Amqp_Tcp_Only", Logger.LoggingLevel.Info);
+                    }
+                    else
+                    {
+                        deviceClient = DeviceClient.CreateFromConnectionString(deviceConnectionStr, TransportType.Mqtt_Tcp_Only);
+                        Logger.Log(DevEUI, $"device client created using Mqtt_Tcp_Only", Logger.LoggingLevel.Info);
+                    }
```

Comment:
same as above

Reply:
Done!

Reply:
- Changed logging levels of the DeviceClient creation to Full.
- Added NET_SRV_LEAF_DEVICE_PROTOCOL environment variable to deployment.*.template.json files and to example.env
- Updated azure-pipeline.yaml to EDGEHUB_MQTTSETTINGS_ENABLED: true